radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/tests/consensus_ledger_sync_epochs/RandomValidatorsTest.java

```diff
@@ -89,20 +86,4 @@ public void given_deterministic_randomized_validator_sets__then_should_pass_bft_
 		TestResults results = bftTest.run();
 		assertThat(results.getCheckResults()).allSatisfy((name, err) -> AssertionsForClassTypes.assertThat(err).isEmpty());
 	}
-
-	@Test
-	public void given_deterministic_randomized_validator_sets_with_incorrect_single_epoch_syncing__then_should_fail() {
```

Comment:
Any reason why this was removed?

Reply:
this doesn't seem to test anything with the new setup